pkg/operator/ceph/provisioner/provisioner.go

```diff
@@ -207,8 +207,8 @@ func parseClassParameters(params map[string]string) (*provisionerConfig, error)
 			cfg.clusterNamespace = v
 		case "fstype":
 			cfg.fstype = v
-		case "datapool":
-			cfg.dataPool = v
+		case "datablockpool":
```

Comment:
for backward compatibility do we actually need to leave `datapool` parsing here? Or was this a new feature in 0.9 so we don't have to worry about that?

Reply:
This was a new feature for v0.9 and the docs always said `dataBlockPool` when v0.9 was released https://github.com/rook/rook/blame/b6b569ba422be38490dd94760ef26358359932cc/cluster/examples/kubernetes/ceph/ec-storageclass.yaml#L33.